source/module_psi/psi.cpp

```diff
@@ -275,11 +302,11 @@ T& Psi<T, Device>::operator()(const int ik, const int ibands, const int ibasis)
     assert(ik >= 0 && ik < this->nk);
 	assert(ibands >= 0 && ibands < this->nbands);	
     assert(ibasis >= 0 && ibasis < this->nbasis);
-    if (this->device != CpuDevice) {
-        #if ((defined __CUDA) || (defined __ROCM))
-        ModuleBase::WARNING_QUIT("Psi operator ()", "GPU psi cannot fetch value by an overloaded operator ()!");
-        #endif 
-    }
+    // if (this->device != CpuDevice) {
+    //     #if ((defined __CUDA) || (defined __ROCM))
+    //     ModuleBase::WARNING_QUIT("Psi operator ()", "GPU psi cannot fetch value by an overloaded operator ()!");
+    //     #endif
+    // }
```

Comment:
It would be better removing obsolete codes.

Reply:
Solved